tests/input_formats/owasp_dependency_scan/services/test_convert_to_opossum.py

```diff
@@ -233,6 +234,56 @@ def _get_attributions(self, root_resource: RootResource) -> list[OpossumPackage]
         )
 
 
+class TestResourceGeneration:
```

Comment:
In these tests, do you also want to check whether the Resource's type is FILE or FOLDER? (It is not clear to me which one would be sensible)
~~Do you want to check `resources_with_children`? IIRC from yesterday, a `virtual` Dependency should be a `file_with_children`?~~ Ah you do it below, seen it now.